src/server/zset_family.cc

```diff
@@ -1340,6 +1342,55 @@ void ZSetFamily::ZInterStore(CmdArgList args, ConnectionContext* cntx) {
   (*cntx)->SendLong(smvec.size());
 }
 
+void ZSetFamily::ZInterCard(CmdArgList args, ConnectionContext* cntx) {
+  unsigned num_keys;
+  if (!absl::SimpleAtoi(ArgS(args, 0), &num_keys)) {
+    return (*cntx)->SendError(OpStatus::SYNTAX_ERR);
+  }
+
+  uint64_t limit = 0;
+  if (args.size() == 1 + num_keys + 2 && ArgS(args, 1 + num_keys) == "LIMIT") {
```

Comment:
can you wrap ` 1 + num_keys + 2` in `()` for clarity? :)

Reply:
Done